src/python/parla/common/spawn.py

```diff
@@ -39,7 +42,11 @@ def closure():
 
 
 # @profile
-def spawn(task=None,  dependencies=[], vcus=1):
+def spawn(task=None,
+          dependencies=[],
+          # TODO(hc): Do we support TaskID? (IIRC, it will be removed?)
```

Comment:
It's already removed. TaskSpaces return a ComputeTask. If the task argument is not a ComputeTask already, it must be a hashable type and we put it in the global taskspace.